optimas/diagnostics/exploration_diagnostics.py

```diff
@@ -54,20 +54,22 @@ def __init__(self, source: Union[str, Exploration]) -> None:
                         "The most recent one will be used."
                     )
                 exploration_dir_path = path
-                output_file = os.path.join(path, output_files[-1])
-                params_file = os.path.join(path, "exploration_parameters.json")
+                self.history_file = os.path.join(path, output_files[-1])
+                self.params_file = os.path.join(
+                    path, "exploration_parameters.json"
+                )
             elif path.endswith(".npy"):
                 exploration_dir_path = pathlib.Path(path).parent
-                output_file = path
-                params_file = os.path.join(
+                self.history_file = path
+                self.params_file = os.path.join(
                     exploration_dir_path, "exploration_parameters.json"
                 )
             else:
                 raise RuntimeError(
                     "The path should either point to a folder or a `.npy` file."
                 )
             exploration = self._create_exploration(
-                exploration_dir_path, params_file, output_file
+                exploration_dir_path, self.params_file, self.history_file
```

Comment:
Is there a need to store the paths to `history_file` and `params_file` in `self`? They are not used anywhere else.

Reply:
You are right, they are not being used anywhere else yet.
Nevertheless, I think that it is important that when one initializes an instance of `ExplorationDiagnostics` from files, it should be known which are those files.
So the user knows exactly where this data is coming from.

Reply:
Right, but the path to the data is already in `exploration_dir_path`. Is there any reason to have the path o the `json` and `npy` files? In principle I don't see a use case.

Reply:
I agree that maybe not the `json` file so much because there is only one, but sometimes there could be more than one history file in the `exploration_dir_path`.
When this is the case, and just the `exploration_dir_path` is given, the most recent history file is automatically selected.
Keeping the path to the selected history file seems to me a good practice so it can be known the origin of the data store in `ExplorationDiagnostics`.